src/riak_pb_codec.erl

```diff
@@ -435,6 +436,23 @@ record_test() ->
 
     decode_eq(Req, encode(Req), fun decode/2).
 
+optional_booleans_test() ->
+    Req = #dtfetchreq{bucket = "bucket",
+                      key = <<"key">>,
+                      type = <<"type">>},
+    DecodedReq = #dtfetchreq{bucket = <<"bucket">>,
+                             key = <<"key">>,
+                             type = <<"type">>,
+                             r = undefined,
+                             pr = undefined,
+                             basic_quorum = undefined,
+                             notfound_ok = undefined,
+                             timeout = undefined,
+                             sloppy_quorum = undefined,
+                             n_val = undefined,
+                             include_context = true},
```

Comment:
This is the important part - ensure that `include_context` default value is honored.